src/plugins/index_pattern_field_editor/public/components/preview/field_preview_empty_prompt.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiEmptyPrompt, EuiText, EuiTextColor, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+export const FieldPreviewEmptyPrompt = () => {
+  return (
+    <EuiFlexGroup style={{ height: '100%' }}>
+      <EuiFlexItem>
+        <EuiEmptyPrompt
+          iconType="inspect"
+          title={
+            <h2>
+              {i18n.translate('indexPatternFieldEditor.fieldPreview.emptyPromptTitle', {
+                defaultMessage: 'Preview',
+              })}
+            </h2>
+          }
+          titleSize="s"
+          body={
+            <EuiText size="s">
+              <EuiTextColor color="subdued">
+                <p>
+                  {i18n.translate('indexPatternFieldEditor.fieldPreview.emptyPromptDescription', {
+                    defaultMessage:
+                      'Configure field name and value or format to see a preview of how new fields will appear.',
```

Comment:
Here's some suggested text for the preview; we can chat about which one works best, or some combination thereof:

- Enter a field name and a script to view a preview of the calculated output
- Enter a field name and a script to display a preview of the generated output
- Enter a field name and a script to view how calculated fields will display

Reply:
Thanks @sebelga -- indeed, it's a tricky situation!

This is the simplest option, though it doesn't describe matching any field names in `_source`. Users could enter `foo.bar`, but if that field doesn't exist in `_source` then they need to add a script:
>Enter a field name or define a script to view a preview of the calculated output

This option provides more clarity around entering the name of an existing field, and is perhaps more accurate of what action users need to take. Users can define a script without a field name, or only enter the name of an existing field:
>Enter the name of an existing field or define a script to view a preview of the calculated output

Reply:
@sebelga, per our discussion, let's go with this option:
>Enter the name of an existing field or define a script to view a preview of the calculated output